deploy/templates/nstemplatetiers/test/ns_dev.yaml

```diff
@@ -197,6 +197,19 @@ objects:
     podSelector: {}
     policyTypes:
     - Ingress
+- apiVersion: networking.k8s.io/v1
```

Comment:
We don't have to modify the test tier. But it won't heart to add it here too.

Reply:
ok, makes sense. I wasn't sure, just wanted to be consitent.